src/app/zap-templates/zcl/data-model/chip/webrtc-provider-cluster.xml

```diff
@@ -0,0 +1,144 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2024 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<!--
+XML generated by Alchemy; DO NOT EDIT.
+Source: src/app_clusters/WebRTC_Provider.adoc
+Parameters: in-progress 
+Git: 0.9-fall2024-234-gf4d359001
+-->
+<configurator xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../zcl.xsd">
+  <domain name="General"/>
```

Comment:
Not sure if this still needs to be CHIP?

Reply:
🤷‍♂️ This doesn't seem to be used anywhere as far as I could tell, i.e. might be vestigial (same for the other one in cluster too, as far as I can tell) - i.e. zap regen was a no-op after these changes